examples/make_eeg_bids.py

```diff
@@ -0,0 +1,199 @@
+# '.edf', which is good for us because next to the BrainVision format, EDF is
+# one of the recommended file formats for EEG BIDS. However, apart from the
+# data format, we need to build a directory structure and supply meta data
+# files to properly *bidsify* this data.
+
+###############################################################################
+# Step 2: Formatting as BIDS
+# --------------------------
+#
+# Let's start by formatting a single subject. We are reading the data using
+# MNE-Python's io module and the `read_raw_edf` function.
+data_dir = os.path.join(data_dir, 'physiobank', 'database', 'eegmmidb')
+edf_path = os.path.join(data_dir, 'S001', 'S001R02.edf')
+raw = read_raw_edf(edf_path, preload=True)
+print(raw)
+
+###############################################################################
+# With this simple step we have everything to start a new BIDS directory using
+# our data. To do that, we can use the high level function `raw_to_bids`, which
+# is the core of MNE-BIDS. Generally, `raw_to_bids` tries to extract as much
+# meta data as possible from the raw data and then formats it in a BIDS
+# compatible way. `raw_to_bids` takes a bunch of inputs, most of which are
+# however optional. The required inputs are:
+#
+# * subject_id
+# * task
+# * raw_file
+# * output_path
+#
+# ... as you can see in the docstring:
+help(raw_to_bids)
+
+###############################################################################
+# We loaded 'S001R02.edf', which corresponds to subject 1 in the second task.
+# The second task was to rest with closed eyes.
+subject_id = '001'  # zero padding to account for >100 subjects in this dataset
+task = 'resteyesclosed'
+raw_file = raw
+output_path = os.path.join(home, 'mne_data', 'eegmmidb_bids')
+
+# Clean up existing directories for the sake of the example
+if os.path.exists(output_path):
+    sh.rmtree(output_path)
+
+###############################################################################
+# Now we just need to specify a few more EEG details to get something sensible:
+
+# First, tell `MNE-BIDS` that it is working with EEG data:
+kind = 'eeg'
+
+# Brief description of the event markers present in the data. This will become
+# the `trial_type` column in our BIDS `events.tsv`. We know about the event
+# meaning from the documentation on PhysioBank
+trial_type = {'rest': 0, 'imagine left fist': 1, 'imagine right fist': 2}
+
+# Now convert our data to be in a new BIDS dataset.
+raw_to_bids(subject_id=subject_id, task=task, raw_file=raw_file,
+            output_path=output_path, kind=kind, event_id=trial_type)
+
+###############################################################################
+# What does our fresh BIDS directory look like?
+print_dir_tree(output_path)
+
+###############################################################################
+# Looks good so far, let's convert the data for all tasks and subjects.
+
+# Start with a clean directory
+sh.rmtree(output_path)
```

Comment:
[again](https://github.com/mne-tools/mne-bids/pull/78/files#r217536493) ?

Reply:
yes, we can remove it once we properly `overwrite`